### PR TITLE
 [heartbeat] add description for survey model and view

### PR DESCRIPTION
**Pull request by lgp171188**
Add a description field to the model making it optional with a default
value of "". Update SurveyAdmin to list the description field. Update
the template used by survey view in the analyzer section to display the
description field

### CODE REVIEW

File: fjord/heartbeat/models.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Survey(ModelBase):
     name = models.CharField(max_length=100, unique=True)
     enabled = models.BooleanField(default=False)
     created = models.DateTimeField(auto_now_add=True)
+    description = models.TextField(blank=True, default="")
```
Comment (willkg) [inline fjord/heartbeat/models.py]: Nit: We use single quotes for strings.

**Comment by lgp171188** [comment]
Fixed the code to use single quotes for string

**Comment by willkg** [comment]
Looks great!

**Comment by willkg** [comment]
1c6eda9 [bug 1122060] add description for heartbeat survey model

Thank you!

**Pull request merged**